cram_pr2/cram_pr2_shopping_demo/src/plans.lisp

```diff
@@ -185,15 +148,22 @@
                            (arm ?grasping-arm)
                            (object ?object)
                            (target (desig:a location
-                                            (pose ?table)))))))
-
+                                            (pose ?destination)))))))
 (defun collect-article ()
   (pr2-proj:with-simulated-robot
-    (let ((objects '(:heitmann :somat :dove :denkmit))
-          object-desigs)
+    (let ((objects '(:heitmann :dove :denkmit))
+          (y 0.2)
+          object-desigs
+          destination)
       (setf object-desigs (try-detecting objects))
       (loop for ?object in object-desigs
-            do (move-object ?object)))))
+            do (setf destination (cl-tf:make-pose-stamped
```

Comment:
Please ``cl-transforms-stamped`` for stamped stuff and ``cl-transforms`` for non stamped stuff:
```lisp
(cl-transforms-stamped:make-pose-stamped
   ...
   (cl-transforms:make-3d-vector ...)
   (cl-transforms:make-identity-rotation))
```

Reply:
5